docker-compose-irods.yml

```diff
@@ -43,231 +22,332 @@ services:
+      ENV_FILEBEAT_PRIMARY_TAG: ICAT
       VIRTUAL_HOST: irods.${RIT_ENV}.rit.unimaas.nl
-      IRODS_INGEST_REMOVE_DELAY: ${ENV_IRODS_INGEST_REMOVE_DELAY}
-      IRODS_TEMP_PASSWORD_LIFETIME: ${ENV_IRODS_TEMP_PASSWORD_LIFETIME}
-      EPICPID_URL: ${ENV_EPICPID_URL}
-      EPICPID_USER: ${ENV_EPICPID_USER}
-      EPICPID_PASSWORD: ${ENV_EPICPID_PASSWORD}
-      MDR_HANDLE_URL: http://mdr.${RIT_ENV}.dh.unimaas.nl/hdl/
-      LOGSPOUT: ignore  #ignore, because logs are forwarded by filebeat (in production it's not in a docker container)
-      DEBUG_DH_BOOTSTRAP:  "no"
     networks:
       default:
         aliases:
-          - irods.dh.local
+          - icat.dh.local
       common_default:
         aliases:
-          - irods.dh.local
+          - icat.dh.local
     volumes:
       - /etc/localtime:/etc/localtime:ro
       - ./externals/irods-ruleset:/rules
       - ./externals/irods-microservices:/microservices
+      - ./externals/dh-irods/icat/test-dev-ssl-certs/:/opt/certificates:ro
+      - ./externals/dh-irods/icat/patch/genquery.py:/etc/irods/genquery.py
+      - ./externals/dh-irods/icat/scripts/bootstrap.sh:/opt/irods/bootstrap.sh
+      - ./externals/dh-irods/icat/scripts/bootstrap_irods.sh:/opt/irods/bootstrap_irods.sh
+      - ./externals/dh-irods/icat/DMFS:/opt/irods/DMFS
       - ./keycloak/users.json:/opt/irods/users.json
-      - microservices-build-ubuntu:/tmp/microservices-build
+      - microservices-build:/tmp/microservices-build
 
-  ires:
+  ires-hnas-um:
     build:
-      context: irods/ires/
+      context: externals/dh-irods/irods-base/
       dockerfile: Dockerfile
       args:
-        - ENV_REGISTRY_HOST        # Used to find base image
-        - ENV_BRANCH               # Used to find base image
-        - ENV_REGISTRY_HOST=${ENV_REGISTRY_HOST}
-        - ENV_BRANCH=${ENV_BRANCH}
+        - ENV_IRODS_VERSION
+        - ENV_IRODS_EXT_CLANG_VERSION
+        - ENV_IRODS_EXT_CLANG_RUNTIME_VERSION
+        - ENV_FILEBEAT_VERSION
+        - ENV_IRODS_RESC_PLUGIN_S3_VERSION
         - ENV_DOCKERIZE_VERSION
-    image: ${ENV_REGISTRY_HOST}/docker-dev/${ENV_BRANCH}/ires:${ENV_TAG}
-    depends_on:
-      - irods
-    hostname: ires.dh.local
+        - ENV_IRODS_GID
+        - ENV_IRODS_UID
+    hostname: ires-hnas-um.dh.local
+    env_file:
+      - .env
+      - irods.secrets.cfg
     environment:
-      RODS_PASSWORD: irods
-      PGPASSWORD: foobar
-      VIRTUAL_HOST: irods.${RIT_ENV}.rit.unimaas.nl
-      IRODS_INGEST_REMOVE_DELAY: ${ENV_IRODS_INGEST_REMOVE_DELAY}
-      IRODS_TEMP_PASSWORD_LIFETIME: ${ENV_IRODS_TEMP_PASSWORD_LIFETIME}
-      EPICPID_URL: ${ENV_EPICPID_URL}
-      EPICPID_USER: ${ENV_EPICPID_USER}
-      EPICPID_PASSWORD: ${ENV_EPICPID_PASSWORD}
-      MDR_HANDLE_URL: http://mdr.${RIT_ENV}.dh.unimaas.nl/hdl/
-      LOGSPOUT: ignore  #ignore, because logs are forwarded by filebeat (in production it's not in a docker container)
-      DEBUG_DH_BOOTSTRAP:  "no"
+      RIT_ENV: dev
```

Comment:
@deanlinssen Did we decide to change this to at the end? Or do we want "RIT_ENV"?

Reply:
ENV_ENVIRONMENT